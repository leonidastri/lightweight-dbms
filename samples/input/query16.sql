select DISTINCT S.A, R.G, B.E, S.C from Sailors S, Boats B, Reserves R WHERE B.D < R.H AND S.A = 1 ORDER BY B.E;