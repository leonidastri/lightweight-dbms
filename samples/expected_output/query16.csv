1,4,1,50
1,1,2,50
1,3,2,50
1,4,2,50
1,4,3,50
1,1,3,50